ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ethernet-interfaces.j2

```diff
@@ -283,6 +283,9 @@
 {%     if port_channel_interface_ipv4.configured == true %}
 *Inherited from Port-Channel Interface
 {%     endif %}
+{# IP NAT Documentation #}
+{%     set ip_nat_interfaces = ethernet_interfaces | arista.avd.default({}) %}
```

Comment:
```suggestion
{%     set ip_nat_interfaces = ethernet_interfaces | arista.avd.default([]) %}
```

Reply:
Validation if `ethernet_interfaces` is defined is already done in line 2. Thus defaulting is not required at all.